documents/entityrelationship/bootstrapper.go

```diff
@@ -0,0 +1,72 @@
+package entityrelationship
+
+import (
+	"github.com/centrifuge/centrifuge-protobufs/documenttypes"
+	"github.com/centrifuge/go-centrifuge/bootstrap"
+	"github.com/centrifuge/go-centrifuge/documents"
+	"github.com/centrifuge/go-centrifuge/errors"
+	"github.com/centrifuge/go-centrifuge/identity"
+	"github.com/centrifuge/go-centrifuge/queue"
+	"github.com/centrifuge/go-centrifuge/storage"
+	"github.com/centrifuge/go-centrifuge/transactions"
+)
+
+const (
+	// BootstrappedEntityHandler maps to grpc handler for entities
+	BootstrappedEntityrelationService string = "BootstrappedEntityrelationService"
+)
```

Comment:
would be nicer if this was BootStrappedEntityRelationshipService (but I can also change in my PR).